tests/console/yast2_clone_system.pm

```diff
@@ -25,7 +25,7 @@ sub run {
     zypper_call "in autoyast2";
 
     my $module_name = y2_module_consoletest::yast2_console_exec(yast2_module => 'clone_system');
-    wait_serial("$module_name-0", 360) || die "'yast2 clone_system' didn't finish";
+    wait_serial("$module_name-0", 1500) || die "'yast2 clone_system' didn't finish";
```

Comment:
let's do an ugly `if aarch` because if this happens in other arch like x86 we would missed some bug for sure.
I can see that normally takes 900 aprox (15 minutes), 1200 could work, right?

Reply:
ok, rerunning the VRs